Gems/MotionMatching/Code/Source/MotionMatchingInstance.cpp

```diff
@@ -448,7 +458,11 @@ namespace EMotionFX::MotionMatching
         float minTrajectoryFutureCost = 0.0f;
 
         // Iterate through the frames filtered by the broad-phase search.
+#ifdef SEARCH_THROUGH_WHOLE_MOTIONDATABASE
```

Comment:
suggestion: Might be worth updating this to use an AZ_CVar so it can be changed at runtime?

Reply:
Good suggestion, I'll get that in along with one of the next PRs